tap_hubspot/__init__.py

```diff
@@ -182,10 +199,16 @@ def get_field_schema(field_type, extras=False):
         }
 
 def parse_custom_schema(entity_name, data):
-    return {
-        field['name']: get_field_schema(field['type'], entity_name != 'contacts')
-        for field in data
-    }
+    if entity_name == "tickets":
+        return {
+            field['name']: get_field_type_schema(field['type'])
+            for field in data["results"]
+        }
+    else:
```

Comment:
No need to put return statement in else block.
Do it like this :-
```
if condition:
    return value1
return value2
```

Reply:
done